### PR TITLE
update repo for mainnet readiness

### PR DESCRIPTION
**Pull request by xenowits**
Update current repo to make it ready for mainnet.

ticket: https://github.com/ObolNetwork/charon-distributed-validator-node/issues/248

### CODE REVIEW

**Comment by gsora** [comment]
Hey @xenowits! 

Is this PR ready for review?

**Comment by xenowits** [comment]
No @gsora

It will be ready by tomorrow

File: prometheus/prometheus.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ remote_write:
         action: keep # Keeps charon metrics and drop metrics from other containers.
 
 scrape_configs:
-  - job_name: "geth"
+  - job_name: "nethermind"
```
Comment (xenowits) [inline prometheus/prometheus.yml:15]: not sure about the metrics path though...

File: docker-compose.yml [hunk 2]
```diff
@@ -194,7 +192,7 @@ services:
      - ./validator-ejector:/exitmessages
    restart: unless-stopped
    environment:
-     -  EXECUTION_NODE=${VE_EXECUTION_NODE_URL:-http://geth:8545}
+     -  EXECUTION_NODE=${VE_EXECUTION_NODE_URL:-http://nethermind:8545}
```
Comment (xenowits) [inline docker-compose.yml:195]: do we need to change `ORACLE_ADDRESSES_ALLOWLIST`?

Comment (gsora) [reply docker-compose.yml:195]: Hmm this is a Lido detail, not really sure, but I believe yes.

You can find them on the validator-ejector github page.

**Review by gsora — APPROVED** [review]

**Pull request merged**